scripts/prompt_history_script.py

```diff
@@ -67,89 +69,109 @@ def add_config(id: str, name: str, model: str, info_text: str, img) -> history.H
             "image": img,
         }
         return
-    
+
     # save image
     if global_state.save_thumbnail:
-        new_width  = 300
+        new_width = 300
         new_height = int(new_width * img.height / img.width)
         img = img.resize((new_width, new_height), Image.LANCZOS)
-        images.save_image_with_geninfo(img, None, os.path.join(global_state.history_path, f"{id}.jpg"))
+        images.save_image_with_geninfo(img, None, os.path.join(

```

Comment:
Not sure what everyone thinking but with me I think this is harder to read. xD

Reply:
vs code `auto-format` here, you can ignore it, and use your prefer format.